htdocs/takepos/invoice.php

```diff
@@ -40,6 +40,10 @@
 $idline = GETPOST('idline');
 $desc = GETPOST('desc', 'alpha');
 $pay = GETPOST('pay');
+$sql="SELECT rowid FROM ".MAIN_DB_PREFIX."facture where ref='(PROV-POS-".$place.")'";
+$resql = $db->query($sql);
```

Comment:
@dbeniamine 
for multicompany compatibility : 

```
$sql = "SELECT rowid FROM ".MAIN_DB_PREFIX."facture";
$sql.= " WHERE entity IN "(".getEntity('takeposinvoice').")";
$sql.= " AND ref='(PROV-POS-".$place.")'";
```

Reply:
This sql request was dead code revived by myself as I ported my contribution to TakePOS9, I removed it.